framework/widgets/ActiveForm.php

```diff
@@ -418,4 +424,49 @@ public static function validateMultiple($models, $attributes = null)
 
         return $result;
     }
+
+    /**
+     * Registers javascript function that toggles 'aria-invalid' attribute
+     * depending on input validation state
+     */
+    protected function registerClientAriaToggle()
```

Comment:
The whole method should be removed. The functionality should be moved to `yii.activeForm.js`